apps/common-app/src/examples/RuntimeTests/tests/animations/withDelay/addDelays.test.tsx

```diff
@@ -0,0 +1,147 @@
+import React, { useEffect } from 'react';
+import { View, StyleSheet } from 'react-native';
+import Animated, { useSharedValue, useAnimatedStyle, withTiming, withDelay } from 'react-native-reanimated';
+import {
+  describe,
+  test,
+  render,
+  useTestRef,
+  getTestComponent,
+  wait,
+  mockAnimationTimer,
+  recordAnimationUpdates,
+  unmockAnimationTimer,
+  expect,
+  clearRenderOutput,
+} from '../../../ReanimatedRuntimeTestsRunner/RuntimeTestsApi';
+
+const WIDTH_COMPONENT_ACTIVE_REF = 'WidthComponentActive';
+const WIDTH_COMPONENT_PASSIVE_REF = 'WidthComponentPassive';
+const WIDTH_COMPONENT_INLINE_REF = 'WidthComponentInline';
+const COMPONENT_REFS = [WIDTH_COMPONENT_ACTIVE_REF, WIDTH_COMPONENT_PASSIVE_REF, WIDTH_COMPONENT_INLINE_REF];
+
+describe('Add three delays', () => {
+  const WidthComponent = ({ delays }: { delays: [number] | [number, number, number] }) => {
+    const widthActiveSV = useSharedValue(0);
+    const widthPassiveSV = useSharedValue(0);
+
+    const refActive = useTestRef(WIDTH_COMPONENT_ACTIVE_REF);
+    const refPassive = useTestRef(WIDTH_COMPONENT_PASSIVE_REF);
+    const refInline = useTestRef(WIDTH_COMPONENT_INLINE_REF);
+
+    const styleActive = useAnimatedStyle(() => {
+      return {
+        width: widthPassiveSV.value,
+      };
+    });
+    const stylePassive = useAnimatedStyle(() => {
+      if (delays.length === 1) {
+        return {
+          width: withDelay(delays[0], withTiming(widthActiveSV.value, { duration: 100 })),
```

Comment:
Have you intentionally used here `widthActiveSV` instead of `widthPassiveSV`?
```suggestion
          width: withDelay(delays[0], withTiming(widthPassiveSV.value, { duration: 100 })),
```

Reply:
Yes, intentionally, I updated styleName to match